docs/ovs-pipeline.md

```diff
@@ -377,34 +377,32 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### CnpEgressRuleTables (45-49)
+### AntreaPolicyEgressRuleTables (45, 49)
 
-For these tables, you will need to keep in mind the CNP [specification](#antrea-network-policy-crd-implementation)
+For these two tables, you will need to keep in mind the Antrea Policy [specification](#antrea-network-policy-crd-implementation)
 that we are using.
 
-These tables are used to implement the egress rules across all ClusterNetworkPolicies.
-Depending on the tier to which the CNP belongs to, the rules will be installed
-in a table corresponding to that tier. The egress CNP table to tier mappings is as
+These two tables are used to implement the egress rules across all ClusterNetworkPolicies
+and Antrea NetworkPolicies.
```

Comment:
nit: using "ClusterNetworkPolicies and Antrea NetworkPolicies" is a bit weird I think
maybe the doc would benefit from using the term "Antrea-native policies" if we define it in the Terminology sections as "Antrea ClusterNetworkPolicies and Antrea NetworkPolicies". Although I notice that https://github.com/vmware-tanzu/antrea/blob/master/docs/network-policy.md never uses the term "Antrea-native policies". @abhiraut 
given the new acronyms (ACNP / ANP), maybe if we don't want to use "Antrea-native policies" to designate both, then we should use "Antrea ClusterNetworkPolicies and Antrea NetworkPolicies" even if it's longer.

Reply:
Updated to "Antrea ClusterNetworkPolicies and Antrea NetworkPolicies" for now. If we are good with "Antrea-native policies" as a term, I can add a new commit to this PR by updating both this doc and network-policy.md.